torchdynamo/variables/user_defined.py

```diff
@@ -281,7 +281,24 @@ def var_getattr(self, tx, name):
         options = VariableTracker.propagate(self)
         value = self.value
         source = AttrSource(self.source, name) if self.source else None
-        self._check_for_getattribute()
+
+        try:
```

Comment:
This is a more general way - however with issues that I am not sure how to resolve.